_installation/single-multiple.md

```diff
@@ -10,13 +10,20 @@ Babelfish offers two modes of operation:
 - `single-db`
 - `multi-db`
 
-What is the purpose of these two modes? In MS SQL, it is really cheap to perform
-joins between tables located in different databases. In PostgreSQL, it is also
-possible to join between databases, but it is far more expensive to do so. Since
-joining different databases is pretty common in MS SQL, this is an important
-issue. The question now is: How can Babelfish model the desired MS SQL behavior?
-In the case of `single-db`, MS SQL databases are mapped to schemas in PostgreSQL. In
-the case of `multi-db`, many databases are used instead of just one.
+When you create a Babelfish cluster, you choose between using a single
+SQL Server database on its own or multiple SQL Server databases together.
+Your choice affects how the names of SQL Server schemas inside the Babelfish
+database appear in PostgreSQL.
+
+If you specify `single-db`, you can create only a single SQL Server database in
+Babelfish, and SQL Server schemas will be created as regular PostgreSQL schemas
+in your Babelfish database.  If you specify `multi-db`, you can create
+several SQL Server databases with schemas in them, and an SQL Server schema will
+be created as PostgreSQL schema `<database name>_<schema_name>` to avoid name
```

Comment:
remove 'with schemas in them' : there are always schemas in a database. THis is just as true for single-db mode as well, BTW

Reply:
'an SQL Server schema' should be 'a SQL Server schema'

Reply:
@rcv-aws I changed the wording to "several T-SQL databases, each with its own schemas" for more exactitude.
@susanmdouglas-aws This is debatable, since there are different received pronunciations of "SQL". But the problem is gone after I renamed it to "T-SQL schema".

Reply:
"SQL Server" is always pronounced as "sequel server", so the article should be 'a'. The word SQL on its own can indeed have different pronunciations, but that is not true for the product name.

Reply:
Got it, thanks for the clarification.